edge-apps/clock/screenly.yml

```diff
@@ -0,0 +1,42 @@
+app_id: ''
+root_asset_id: ''
+user_version: 'v1.0.0'
+revision: 1
+description: 'Screenly Clock Edge App'
+icon: ''
+author: 'Screenly, Inc.'
+homepage_url: ''
+settings:
+  override_timezone:
+    type: text
+    default: "false"
+    optional: true
+    title: Override Timezone
+    help_text: |
+      Whether to override the timezone with the timezone specified
+      in settings.
+  timezone:
+    type: text
+    default: '' # Can we omit `default` if the field's not optional?
+    optional: false
```

Comment:
This is actually wrong. We shouldn't require the setting of a timezone. The setting should be optional. The default value should be the *location* of the screen. Meaning that we will need to find the timezone based on the location (using lat/lng from metadata).